### ISSUE TITLE
Migration to go.mod and module version

### ISSUE THREAD

**Issue by combor** [EngineerBetter/stopover#7]
Hi, thanks for merging https://github.com/EngineerBetter/stopover/pull/6 so quickly.

We tried to install v2.0.1 using `go get/install` but unfortunately it complains about module versioning:
```
go install -v github.com/EngineerBetter/stopover/v2@v2.0.1
go install github.com/EngineerBetter/stopover/v2@v2.0.1: github.com/EngineerBetter/stopover@v2.0.1: invalid version: module contains a go.mod file, so major version must be compatible: should be v0 or v1, not v2
```
We think the reason may be that the module was migrated to use `gomod` but only the minor version was incremented in the git tag. The module docs mention this:

_Note that if you are adopting modules for the first time for a pre-existing repository or set of packages that have already been tagged v2.0.0 or higher before adopting modules, then the recommended best practice is to increment the major version when first adopting modules._

Basically `go install` doesn't recognize the version as coming from go mod and is using the older versioning mechanism as the previous v2.0.0 exists and doesn't have go.mod 
More info can bee found here: 
https://github.com/golang/go/wiki/Modules#releasing-modules-v2-or-higher

They also mention some alternative methods of migration that may be preferable if v3 is not desirable.

Would you be able to consider picking one of the migration options? If alternative methods are preferable we are keen to provide PR.
